config/v1/types_feature.go

```diff
@@ -131,11 +131,33 @@ type FeatureGateAttributes struct {
 	// +required
 	Name FeatureGateName `json:"name"`
 
+	// RequiredMinimumKubeletVersion is the lowest version the MinimumKubeletVersion field in the
+	// nodes.config object may be set to to enable this feature.
+	// +kubebuilder:validation:MaxItems:=1
+	// +listType=map
+	// +listMapKey=component
+	// +openshift:enable:FeatureGate=MinimumKubeletVersion
+	// +optional
+	RequiredMinimumComponentVersions []RequiredMinimumComponentVersion `json:"requiredMinimumComponentVersions,omitempty"`
+
 	// possible (probable?) future additions include
 	// 1. support level (Stable, ServiceDeliveryOnly, TechPreview, DevPreview)
 	// 2. description
 }
 
+type RequiredMinimumComponentVersion struct {
+	Component RequiredMinimumComponent `json:"component"`
+	// +kubebuilder:validation:XValidation:rule="self == \"\" || self.matches('^[0-9]*.[0-9]*.[0-9]*$')",message="minmumKubeletVersion must be in a semver compatible format of x.y.z, or empty"
+	// +kubebuilder:validation:MaxLength:=8
+	// +openshift:enable:FeatureGate=MinimumKubeletVersion
+	Version string `json:"version"`
+}
+
+// +kubebuilder:validation:Enum:=kubelet
```

Comment:
Enums should be PascalCase
```suggestion
// +kubebuilder:validation:Enum:=Kubelet
```